Dockerfile

```diff
@@ -1,7 +1,12 @@
+FROM alpine:latest as certs
+RUN apk --update add ca-certificates
+
 FROM scratch
 
 LABEL maintainer="Torin Sandall <torinsandall@gmail.com>"
 
+COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
```

Comment:
Yeah, this was the first time I've worked with the `scratch` image.. took _quite_ some time to understand why I'd get certificate verification errors 😄 I definitely wouldn't mind swapping the base image for something else, but since I got this working now I don't want to introduce too many new changes in a single PR.